br/pkg/lightning/backend/external/iter.go

```diff
@@ -68,11 +69,13 @@ func (h *mergeHeap[T]) Pop() interface{} {
 }
 
 type mergeIter[T heapElem, R sortedReader[T]] struct {
-	h             mergeHeap[T]
-	readers       []*R
-	curr          T
-	lastReaderIdx int
-	err           error
+	h                mergeHeap[T]
+	readers          []*R
+	curr             T
+	lastReaderIdx    int
+	err              error
+	checkHotPointMap map[int]int
```

Comment:
Maybe rename to `hotspotMap`?

Reply:
I didn't understand the meaning of this name at first glance too.